tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -315,6 +315,18 @@ functionCall(
             ,
                 ['ensure_fully_multiline' => true],
             ],
+            'test wrongly formatted half-multiline function becomes fully-multiline' => [
+                '<?php
+f(
+    1,
+    2,
+3
```

Comment:
Shouldn't this line be indented as well?

Reply:
this rule is not changing indentation for arguments that are already on separated line, see `test ungodly code` case.
if we want to add it, it's ouf ot the scope of this fix

Reply:
Damn it, I always forget that for this fixer! :D